EvanBancroft/portfolio
# Added Copy, Images, and Footer

# Description

Replaces Lorem Ipsum with final copy and adds all content for the 4 initial works. 

Added link to download resume from the skills section. 

Added footer with links to Github, site repo, and LinkedIn. 


# Considerations

- [x] App successfully builds w/ `gatsby build`

# How to test

1. Make sure all the content looks good.
1. Make sure all the new links work. 


## Screenshots (if applicable)
